### PR TITLE
NO-ISSUE: download kubeconfig in BaseKubeAPI test

### PR DESCRIPTION
**Pull request by danielerez**
On BaseKubeAPI test, download kubeconfig using secret ref in ACI.

### CODE REVIEW

File: src/tests/base_kubeapi_test.py [hunk 1]
```diff
@@ -125,6 +125,7 @@ def _wait_for_install(cls, agent_cluster_install: AgentClusterInstall, agents: L
         agent_cluster_install.wait_to_be_installing()
         Agent.wait_for_agents_to_install(agents)
         agent_cluster_install.wait_to_be_installed()
+        agent_cluster_install.download_kubeconfig()
```
Comment (osherdp) [inline src/tests/base_kubeapi_test.py:128]: isn't ``kubeconfig_path`` a required argument?

**Review by osherdp — CHANGES_REQUESTED** [review]

File: src/tests/base_kubeapi_test.py (hunk 1 above)
Comment (danielerez) [reply src/tests/base_kubeapi_test.py:128]: oh right, assumed it's the same as cluster.download_kubeconfig: https://github.com/openshift/assisted-test-infra/blob/6aaf9a3f08d7e9f17ff281c8cfdf29f55a8a5b9d/src/assisted_test_infra/test_infra/helper_classes/cluster.py#L908

**Comment by danielerez** [comment]
/unhold

addressing cluster connection issue on: https://github.com/openshift/assisted-test-infra/pull/1850

```
INFO[2022-09-18T16:17:23Z] ************ vsphere assisted test-infra post-install ************
Could not create directory '/.ssh' (Permission denied).
Unable to connect to the server: dial tcp: lookup api.ci-op-dbmts2jk-9d0ef.redhat.com on 172.30.0.10:53: no such host
```

**Review by osherdp — APPROVED** [review]

**Pull request merged**